docker-compose.yaml

```diff
@@ -55,6 +60,10 @@ services:
           - 8000:8000
         command: bash -c "python manage.py migrate && python manage.py bootstrap && python manage.py runserver 0.0.0.0:8000"
     client:
+        labels:
+          - "traefik.enable=true"
+          - "traefik.http.routers.client.rule=Host(`muscle.local`)"
+          - "traefik.http.routers.client.entrypoints=web"
```

Comment:
What does this setting do? What is the "web" entrypoint?

Reply:
`web` in this context means insecure http traffic, which is fine in a development environment context. If this were to be a production configuration we might want to use `websecure` instead of `web`.